Cargo.toml

```diff
@@ -5,6 +5,8 @@ authors = ["Karsten Jeschkies <jeskar@web.de>"]
 
 [dependencies]
 bytes = "0.4"
+failure = "0.1.1"
```

Comment:
Not sure if it is a good practice, but I normally do not specify the fix version, since probably you want also to get fixes.